Source/WebKit/Shared/WebCoreArgumentCoders.cpp

```diff
@@ -729,6 +730,27 @@ bool ArgumentCoder<ResourceError>::decode(Decoder& decoder, ResourceError& resou
     return true;
 }
 
+void ArgumentCoder<UserAgentStringQuirk>::encode(Encoder& encoder, const UserAgentStringQuirk& userAgentStringQuirk)
```

Comment:
I think this should go in WebCoreArgumentCoders.serialization.in

Reply:
Good call